exercises/reverse-string/README.md

```diff
@@ -0,0 +1,42 @@
+# Reverse String
+
+Reverse a string
+
+For example:
+input: "cool"
+output: "looc"
+
+# Hints
+
+Ruby documentation is pretty extensive! Look through Ruby's built-in `String` methods and see if you can find something that will help you out with this exercise.
```

Comment:
```suggestion
The Ruby library is pretty extensive! Look through Ruby's built-in `String` methods and see if you can find something that will help you out with this exercise.
```
It's more the language than the documentation that's extensive. Can you live with this?